pkg/controllers/networkinfo/networkinfo_controller.go

```diff
@@ -501,20 +486,12 @@ func (r *NetworkInfoReconciler) deleteVPCsByName(ctx context.Context, ns string)
 	return r.deleteVPCs(ctx, vpcToDelete, ns)
 }
 
+// deleteVPCsByID is called when the NetworkInfo CR still exists in the K8s api-server, and the CR's
+// deletionTimestamp is not 0. In this case, it's Namespace must exist, and no other Namespaces configured
+// with the same name. So we can directly use the Namespace's name to find the stale VPCs to delete.
```

Comment:
seems id param is not used, could we remove it?

Reply:
Do we need consider that NetworkInfo CR is deleted by accidently but the namespace still there?
In this case we should not delete VPCs?

Reply:
What do you think if I remove function `deleteVPCsByID`, and call `deleteVPCsByName` instead? Does it resolve your concern?

Reply:
yes, deleteVPCsByName should be fine.